src/packages/@ncigdc/modern_components/ClinicalAnalysis/ContinuousAggregationQuery.js

```diff
@@ -255,11 +255,12 @@ export default compose(
     }
   ),
 )(({
-  aggData, hits, isLoading, setId, stats, ...props
+  aggData, hits, isLoading, key, setId, stats, ...props
```

Comment:
this is going to get removed in the PR for PRTL-2917

Reply:
this was updated in "Merge branch 'feat/continuous-card-refactor' into prtl-2915"

Reply:
resolved by merging PRTL-2917